event_routing_backends/tasks.py

```diff
@@ -82,5 +82,57 @@ def send_event(task, event_name, event, router_type, host_config):
             ),
             exc_info=True
         )
+        raise task.retry(exc=exc, countdown=getattr(settings, 'EVENT_ROUTING_BACKEND_COUNTDOWN', 30),
+                         max_retries=getattr(settings, ''
+                                                       'EVENT_ROUTING_BACKEND_MAX_RETRIES', 3))
```

Comment:
Uck.. this is more of a whinge about Ralph, but.. if even one of the events in the batch we send is a duplicate, then the whole request fails with a 400 Bad Request error. For a command that will likely need to be re-entrant, this is really annoying.

Reply:
I actually wrote that code based on my understanding of the xAPI spec at the time. I think it was a flawed understanding, though, and I'll see if they will take an update to that logic to not fail on duplicate statements unless the same event id in the same batch (which is a bad thing and should fail). Spec is here: https://github.com/adlnet/xAPI-Spec/blob/1.0.3/xAPI-Communication.md#212-post-statements

Reply:
That would be fantastic.

Reply:
Created an issue for it. It turns out I didn't write it, just copied it to the PUT implementation where it is probably more appropriate: https://github.com/openfun/ralph/issues/345

Reply:
Thank you!